### ISSUE TITLE
Websocket - start/stop behaviour

### ISSUE THREAD

**Issue by Kamtsa** [sammchardy/python-binance#26]
On v0.4.1: Websockets in API overall work fine - edge case behaviour starting/stopping is unexpected for me. (I'm unfamiliar with websocket usage, possibly I'm using it wrongly?)

For orderbook depth sockets:
* existing sockets don't stop with bm.stop_socket(conn_key) , only with bm.close() [which stops all sockets]
* after calling bm.close() websockets can't be re-started with bm.start()
* the update speed on the 'default' setting is irregular, providing parameter update_time=WEBSOCKET_UPDATE_1SECOND makes it regular and approx. to 1000ms, maybe the default is different from 1sec? (Using WEBSOCKET_UPDATE_0SECOND myself)
*  in basic example provided: should it read 'e' instead of e (to allow copy and pasting)?
# start trade websocket
def process_message(msg):
    print("message type:" + msg[e])
    print(msg)
    # do something

**Comment by sammchardy** [comment]
Hi @Kamtsa 

v0.4.3 should address these points.

Sockets now stop correctly when calling bm.stop_socket(conn_key), they were reconnecting after that call.
I've updated the bm.close() call to close all open sockets. Sockets can now be restarted after that call.
The update speed parameter is now no longer supported as of v0.4.2, Binance only allow the default option which used to be 1000ms and I can't guarantee what it actually is now.
Fixed documentation as well thanks!
